src/main/scala/org/ua/ilm/akka/ParallelActorMain.scala

```diff
@@ -0,0 +1,33 @@
+package org.ua.ilm.akka
+
+import akka.actor.{Actor, ActorRef, ActorSystem, Props}
+import akka.routing.BalancingPool
+import org.ua.ilm.Entity.Channel
+import org.ua.ilm.akka.ParallelActorMain.listChannels
+
+import scala.collection.mutable.ListBuffer
+
+object ParallelActorMain {
+  val listChannels: ListBuffer[Channel] = ListBuffer.empty
+
+  def parallelCrawlerChannels(key: String, arrIdChannels: Array[String],
```

Comment:
You are composing actor system start and storage actor in one entity - separate them

Reply:
Also I would like you to run all your system by timer, like batching and shutdown by end signal. There is no need to terminate it after timeout